filament/backend/src/vulkan/VulkanDriver.cpp

```diff
@@ -1370,10 +1372,6 @@ void VulkanDriver::commit(Handle<HwSwapChain> sch) {
 
     VulkanSwapChain* swapChain = mResourceAllocator.handle_cast<VulkanSwapChain*>(sch);
 
-    if (mCommands->flush()) {
-        collectGarbage();
-    }
```

Comment:
wouldn't make more sense to flush here as opposed to in endFrame? e.g. on GL eglSwapBuffers() makes an implicit flush.

Reply:
yes, it's been moved inside present